packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -60,26 +61,18 @@ export const WriteOnlyFunctionForm = ({
       toast.error(message);
     }
 
-    if (writeAsync) {
+    if (writeAsync && writeTxn) {
       try {
         // ToDo. Use a transactor component.
-        await reactHotToast.promise(
-          writeAsync(),
-          {
-            loading: "Mining transaction, Hold tight!",
-            success: "Mined successfully !",
-            error: "Error while processing the transaction",
-          },
-          {
-            success: {
-              icon: "🔥",
-            },
-          },
-        );
+        await writeTxn(writeAsync());
         setRefreshDisplayVariables(prevState => !prevState);
       } catch (e: any) {
         const message = getParsedEthersError(e);
-        toast.error(message);
+        toast.error(message, {
+          style: {
+            wordBreak: "break-all",
+          },
+        });
```

Comment:
This is using our custom toast, which only take 1 param. This shouldn't be working.

Reply:
Yes, sorry my bad